packages/turbo-repository/rust/src/internal.rs

```diff
@@ -67,13 +83,22 @@ impl Workspace {
                 path: workspace_state.root.clone(),
             })?
             .clone();
```

Comment:
We don't need the clone here as `PackageManager` implements `Copy` which means we can just dereference a `&PackageManager` instead of cloning it to get a `PackageManager`. See https://rust-lang.github.io/rust-clippy/master/index.html#/clone_on_copy

Reply:
The clone() is chained to `workspace_state`, not `package_manager`, if I'm reading the correctly. if I remove the clone, the compiler complains because workspace_state is also returned from this method. In any case (outside my diff, so I won't change this right now)